crates/bevy_state/src/state_scoped.rs

```diff
@@ -52,7 +53,7 @@ use crate::state::{StateTransitionEvent, States};
 /// app.enable_state_scoped_entities::<GameState>();
 /// app.add_systems(OnEnter(GameState::InGame), spawn_player);
 /// ```
-#[derive(Component)]
+#[derive(Component, Reflect, Clone)]
```

Comment:
Nit: we could probably also add a `ReflectComponent` registration so this component is visible in editors and accessible via reflection

Reply:
Good idea. I'll add that.